torch/csrc/jit/fuser/codegen.cpp

```diff
@@ -397,13 +463,26 @@ std::string generateKernel(
         env.s("access", format("__ldg(&t${formal}.data[t${formal}_offset])", env));
       } else {
         env.s("access", format("t${formal}.data[t${formal}_offset]", env));
+        if (vectorizable) {
+          env.s("vector_access", format("Vec256::load8(t${formal}.data + t${formal}_offset)", env));
+        }
       }
       env.s("lhs_type", calcScalarTypeName(input.second.value().scalar_type));
+      if (vectorizable) {
+        env.s("vector_lhs_type", "Vec256");
+      }
     } else {
       env.s("access", format("s${formal}", env));
       env.s("lhs_type", variableType(input.first->type()));
+      if (vectorizable) {
+        env.s("vector_access", format("Vec256(s${formal})", env));
+        env.s("vector_lhs_type", "Vec256");
+      }
     }
     body << format("${lhs_type} ${node} = ${access};\n", env);
+    if (vectorizable) {
```

Comment:
probably don't need to guard this stuff as long as you are writing to vector-specific strings anyway.